### ISSUE TITLE
Can eslint use a global `.eslintrc` file?

### ISSUE THREAD

**Issue by littlee** [eslint/eslint#3677]
Can eslint use a global file?
when I create a new project, I have to run `eslint --init` in each project,
how do I let eslint use a global `.eslintrc` file?

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by btmills** [comment]
If eslint doesn't find a `.eslintrc` anywhere in your project, it will look for one in your home directory (`~/.eslintrc` or `C:\Users\username\.eslintrc`).

**Comment by jurajpetrik** [comment]
I was having an issue with eslint seemingly ignoring my `~/.eslintrc` and it looks like eslint searches recursively upwards the path tree until it finds `.eslintrc`.
My issue was that I was running eslint from `~/src/project/` and had placed an `.eslintrc` file in `~/src` by accident. which was overriding my config file in the home directory.
I hope this helps someone.

**Comment by Thaina** [comment]
Are there anyway we can init eslint at home directory?

**Comment by NTMan** [comment]
Could you please describe this behavior in documentation?
http://eslint.org/docs/user-guide/configuring#using-configuration-files

**Comment by platinumazure** [comment]
@NTMan It's documented in this section: http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

If you believe it can be improved, please feel free to open a pull request.

**Comment by dingo-d** [comment]
What if I have installed eslint globally (`npm install -g eslint`) and now it sits in my `AppData\Roaming\npm\node_modules\eslint` folder (I'm on Windows 10), and I want to lint file that is on another disk (or mapped server drive)? 

Since the configuration cascades to the root folder if the `.eslintrc` file isn't found in a project, it will only look to the root of the file disk. So if I installed eslint on my laptop, but I'd like to lint a file that is on the work server, I need to place `.eslintrc` file in the root of the mapped disk for it to work. Is there a workaround this? A way to specify the location of that file?
